lib/ClientGeneratorFromXml.php

```diff
@@ -350,7 +353,7 @@ public function getParam($key)
 		return $this->_params[$key];
 	}
 	
-	protected function addFile($fileName, $fileContents, $addLicense = true)
+	private function addFile($fileName, $fileContents, $addLicense = true)
```

Comment:
addFile cannot be private since it is being called by all the derived classes beside testmeDoc